src/molecules/Chip.tsx

```diff
@@ -1,34 +1,52 @@
 import React from 'react';
-import { StyleSheet, View, ViewStyle, Text } from 'react-native';
+import { FlexStyle, StyleProp, StyleSheet, View, ViewStyle, Text } from 'react-native';
 import { registeredTextStyle } from 'src/utils/textStyles';
 import { colors } from 'src/utils/colors';
 
 const styles = StyleSheet.create({
   Wrapper: {
-    alignSelf: 'flex-start', // the only way to make sure that the element won't grow
     paddingHorizontal: 4,
     borderRadius: 7,
     height: 14,
     backgroundColor: colors.tertiary, // default bgState maps to this
   } as ViewStyle,
-  Text: registeredTextStyle('micro', colors.ulisse, 'chip-text'),
 });
 
 export type ChipProps = {
   label: string;
-  bgState: 'default' | 'success' | 'error';
+  bgState: 'default' | 'success' | 'error' | 'ulisse' | 'brand';
```

Comment:
this change makes `bgState` as a name no longer as meaningful. A _state_ of `ulisse` doesn't mean much to me, at least :) 

This should be refactored to `bgColor`, renaming `default` to `none`.

Reply:
Solved by creating additional prop bgColor